rpcs3/Emu/Cell/Modules/cellGcmSys.cpp

```diff
@@ -1123,7 +1123,7 @@ s32 cellGcmSetDefaultCommandBufferAndSegmentWordSize(u32 bufferSize, u32 segment
 	const auto& put = vm::_ref<CellGcmControl>(gcm_info.control_addr).put;
 	const auto& get = vm::_ref<CellGcmControl>(gcm_info.control_addr).get;
 
-	if (put != 0x1000 || get != 0x1000 || bufferSize < segmentSize * 2)
+	if (put != 0x1000 || get != 0x1000 || bufferSize < segmentSize * 2 || segmentSize * 2 < segmentSize)
```

Comment:
I guessed that but you should make intent clearer. Just check if segmentSize >= 0x80... Its more readable  that way and performance is actually better.